EncounterMe/EncounterMe/Views/AllObjectsPage.xaml.cs

```diff
@@ -110,6 +110,11 @@ async void More_Info_Clicked(object sender, EventArgs e)
             await Shell.Current.Navigation.PushAsync(new IndividualObjectPage(pinToPass));
         }
 
+        void Favourite_clicked(object sender, EventArgs e)
```

Comment:
Bet mes dabar niekur nesaugom jo, taip?

Reply:
Niekur, čia palikau, kad būtų aiškiau kaip iš to mygtuko mappin gaut 😄

Reply:
Good 😄